ci/infra/openstack/master-instance.tf

```
@@ -64,10 +64,10 @@ resource "openstack_compute_instance_v2" "master" {
     name = "${var.internal_net}"
   }
 
-  security_groups = [
+  security_groups = "${var.enable_openstack_port_security == true ? [
```

I think this is missing the closing `{`
```suggestion
  security_groups = "${var.enable_openstack_port_security} == true ? [
```